tests/resolvers/Query/getVolunteerMembership.spec.ts

```
@@ -0,0 +1,143 @@
+import type { TestUserType } from "../../helpers/user";
+import { createVolunteerAndActions } from "../../helpers/volunteers";
+import type { InterfaceVolunteerMembership } from "../../../src/models";
+import { VolunteerMembership } from "../../../src/models";
+import { getVolunteerMembership } from "../../../src/resolvers/Query/getVolunteerMembership";
+
+export enum MembershipStatus {
+  INVITED = "invited",
+  REQUESTED = "requested",
+  ACCEPTED = "accepted",
+  REJECTED = "rejected",
+}
```

Biome

[error] Do not export from a test file.



(lint/suspicious/noExportsInTest)